0053-bundle-format/README.md

```diff
@@ -0,0 +1,484 @@
+- `--trusted-root` - Optional path to TrustedRoot JSON file
+
+**Hardware Security:**
+- `--sk` - Use a hardware security key
+- `--slot` - Security key slot to use (default: `signature`)
+
+##### `zarf package verify` Flags
+
+The following flags from `cosign verify-blob` will be available:
+
+**Key & Certificate Verification:**
+- `--key` - Path to public key file, KMS URI, or Kubernetes Secret
+- `--certificate` - Path to public certificate
+- `--certificate-chain` - CA certificates in PEM format for chain building
+- `--ca-roots` - Bundle of CA certificates for certificate chain
+- `--ca-intermediates` - Intermediate CA certificates in PEM format (used with `--ca-roots`)
+
+**Certificate Identity Verification:**
+- `--certificate-identity` - Expected identity in Fulcio certificate
+- `--certificate-identity-regexp` - Regular expression alternative for identity matching
+- `--certificate-oidc-issuer` - Expected OIDC issuer in certificate
+- `--certificate-oidc-issuer-regexp` - Regular expression alternative for issuer matching
+
+**Transparency Log & Timestamp Verification:**
+- `--rekor-url` - Address of Rekor transparency log server (default: `https://rekor.sigstore.dev`)
+- `--timestamp-certificate-chain` - PEM-encoded certificate chain for RFC3161 timestamp authority
+
+**Trust:**
+- `--trusted-root` - Path to Sigstore TrustedRoot JSON file
+- `--private-infrastructure` - Skip transparency log verification for private deployments
+
+**Hardware Security:**
+- `--sk` - Use hardware security key
+- `--slot` - Security key slot (authentication, signature, card-authentication, key-management)
+
+### Trusted Root Management
+
+#### Embedded Trusted Root Update Process
+
+The embedded trusted root should be updated regularly to include new Sigstore keys and prevent expiration issues. Zarf will utilize The Update Framework (TUF) to establish provenance in the retrieval and update process for the embedded data.
+
+**Update Schedule**:
+- Checked before releases
+- Automatically with tooling
+
+#### Embedded Trusted Root SDK Experience
+
+The Zarf Cosign utilities in the SDK offer generic implementation around `sign-blob` and `verify-blob`. The embedded trusted root is available for consumption by default while also allowing SDK consumers to override with their own embedded trusted root by passing it as a referenced path. 
+
+### Migration Path from Legacy Format
+
+#### Phase 1: Dual Format Support
+
+**Behavior**:
+- Both `zarf.bundle.sig` and `zarf.yaml.sig` created during signing
+- Verification checks bundle first, falls back to legacy
+- Deprecation warnings logged when legacy format is used
+- All new packages signed with bundle format
+
+**Actions**:
+- Document bundle format in user tutorials
+- Add migration guide to documentation
+- Provide tooling to re-sign pre-existing packages signed with legacy format signature
+
+#### Phase 2: Bundle Format Default, Legacy Deprecated
+
+**Behavior**:
+- Only `zarf.bundle.sig` created during signing
```

Comment:
I know a file will be required for portability (i.e. the signature bundle should move along with a Zarf Package archive) but wanted to also bring up from some other discussions that it may be worth considering how this is represented in an OCI registry as well.

For example as more projects integrate with the cosign ecosystem it may be nice to be able to use a normal `cosign verify` without needing Zarf at all to normalize tooling and the integration points for other products.  I think that could maybe be as easy as storing the signature bundle in the registry on a `zarf package publish` in the same format as a `cosign upload`.

Reply:
cc @joonas